feature-libs/product/future-stock/components/future-stock-accordion/future-stock-accordion.component.html

```diff
@@ -0,0 +1,21 @@
+<button class="cx-future-stock-accordion-header" (click)="toggle()">
+  <span>
+    {{header}}
+    <cx-icon *ngIf="expanded" [type]="iconType.CARET_UP"></cx-icon>
+    <cx-icon *ngIf="!expanded" [type]="iconType.CARET_DOWN"></cx-icon>
+  </span>
+</button>
+
+<div *ngIf="expanded">
+  <div *ngIf="!isString(content); else contentString">
+    <div class="cx-future-stock-accordion-content" *ngFor = "let elem of content">
+      {{elem.formattedDate}} - {{'futureStockTrigger.quantity' | cxTranslate}} {{elem.stock.stockLevel}}
+    </div>
+  </div>
+
+  <ng-template #contentString>
+    <div class="cx-future-stock-accordion-content">
+      {{content}}
```

Comment:
As I mentioned nice-to-have would be if this content could be also translated.
Additionally, noticed that template styles, indentation, empty line EOF etc. looks different than in the entire Spartacus code.

Can you run: `yarn prettier:fix` to verify? Also please check if the prettier plugin in your IDE works properly.

Reply:
done